src/kernl/implementations/README.md

```diff
@@ -0,0 +1,41 @@
+# Naming conventions
```

Comment:
README in a more visible location like contribution folder? With a good title / name so we know its purpose.